catalog.html

```diff
@@ -0,0 +1,229 @@
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Каталог</title>
+  </head>
+  <body>
+    <header class="page-header">
+
+      <a href="#"><img class="logo" src="#" alt="Логотип интернет-магазина девайсов»."></a>
+
+      <div>
+
+       <p class="navigation-item">
+          <a class="navigation-link" href="#">Войти</a></p>
+      </div>
+
+      <ul class="navigation-list navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">Сравнить</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">Корзина</a>
+        </li>
+      </ul>
+
+      <nav class="navigation">
+        <p>Каталог товаров</p>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="№">Доставка</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="№">Гарантия</a>
+          </li>
+           <li class="navigation-item">
+            <a class="navigation-link" href="№">Контакты</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+<main class="main-inner">
+  <header class="inner-header">
+    <h1 class="inner-header-title">Моноподы для селфи</h1>
+    <ul class="breadcrumbs">
+      <li class="breadcrumbs-item">
+        <a class="breadcrumbs-link" href="index.html">Главная</a>
+      </li>
+      <li class="breadcrumbs-item">
+        <a class="breadcrumbs-link">Каталог товаров</a>
+      </li>
+      <li class="breadcrumbs-item">
+        <a class="breadcrumbs-link" href="#">Моноподы для селфи</a>
+      </li>
+    </ul>
```

Comment:
```suggestion
    <ul class="breadcrumbs">
      <li class="breadcrumbs-item">
        <a class="breadcrumbs-link" href="index.html">Главная</a>
      </li>
      <li class="breadcrumbs-item">
        <a class="breadcrumbs-link" href="#">Каталог товаров</a>
      </li>
      <li class="breadcrumbs-item">
        <a class="breadcrumbs-link" href="catalog.html" aria-current="page">Моноподы для селфи</a>
      </li>
    </ul>
```
Судя по макету, мы находимся не в главном каталоге, а уже в разделе Моноподы. 
Можно:
1. Убрать ссылку вообще
2. Заменить `a` на `span`
3. Убрать href, оставив ссылку
4. Оставить href и добавить `aria-current="page"` чтобы указать, что это ссылка ведет сама на себя. 

Аналогично для пунктов в меню.